### PR TITLE
feat: added with-thirdweb example

### PR DESCRIPTION
**Pull request by avneesh0612**
I have added 2 example repositories for [thirdweb](https://thirdweb.com)

### CODE REVIEW

File: examples/with-thirdweb-typescript/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (balazsorban44) [inline examples/with-thirdweb-typescript/LICENSE.md:1]: Our examples do not have a LICENSE, I'm curious why this was added here?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
We prefer TypeScript for our examples, so could you remove `with-thirdweb` and rename `with-thirdweb-typescript` to `with-thirdweb` instead?

File: examples/with-thirdweb-typescript/LICENSE.md (hunk 1 above)
Comment (avneesh0612) [reply examples/with-thirdweb-typescript/LICENSE.md:1]: Oh my bad. I will remove the licenses

**Comment by avneesh0612** [comment]
@balazsorban44 gotchya

File: examples/with-thirdweb/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@thirdweb-dev/react": "^2",
+    "@thirdweb-dev/sdk": "^2",
+    "ethers": "^5.6.1",
+    "next": "12.1.0",
```
Comment (balazsorban44) [inline examples/with-thirdweb/package.json:15]:
```suggestion
    "next": "latest",
```

File: examples/with-thirdweb/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```
Comment (balazsorban44) [inline examples/with-thirdweb/package.json:17]:
```suggestion
    "react": "18.0.0",
    "react-dom": "18.0.0"
```

File: examples/with-thirdweb/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "devDependencies": {
+    "@types/node": "17.0.21",
+    "@types/react": "17.0.40",
+    "eslint": "8.11.0",
+    "eslint-config-next": "12.1.0",
```
Comment (balazsorban44) [inline examples/with-thirdweb/package.json:23]:
```suggestion
    "eslint-config-next": "latest",
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Let's make sure dependencies are up-to-date as of writing.

**Comment by avneesh0612** [comment]
alright just checking, and doing yarn install again so the lock is in sync

**Comment by balazsorban44** [comment]
I'm not seeing the images being used anywhere. Should we add them somewhere or just remove them?

**Comment by avneesh0612** [comment]
I think we can add one of them as a favicon. What do you think? @balazsorban44

**Review by chaqchase — DISMISSED** [review]

**Review by balazsorban44 — DISMISSED** [review]
All in all, looks good, we can get it merged I think. :+1:

**Comment by avneesh0612** [comment]
okay cool! :) @balazsorban44

File: examples/with-thirdweb/package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "next-typescript-starter",
+  "version": "0.1.0",
```
Comment (ijjk) [inline examples/with-thirdweb/package.json:3]:
```suggestion
```
These fields should be omitted for examples which is why this check is currently failing https://github.com/vercel/next.js/runs/6273678404?check_suite_focus=true

**Pull request closed without merging**